pages/common/bower.md

```diff
@@ -0,0 +1,32 @@
+# bower
+
+> A package manager optimized for the front-end web development.
```

Comment:
I was going to disagree with @igorshubovych but then I tried to find a reference somewhere of front-end development referring to something else than the web (html/css/js) but I couldn't. 

So, could it really be that confusing? What's our final choice?